test/src/rabbit_mqtt_test_util.erl

```diff
@@ -0,0 +1,59 @@
+%% The contents of this file are subject to the Mozilla Public License
+%% Version 1.1 (the "License"); you may not use this file except in
+%% compliance with the License. You may obtain a copy of the License
+%% at http://www.mozilla.org/MPL/
+%%
+%% Software distributed under the License is distributed on an "AS IS"
+%% basis, WITHOUT WARRANTY OF ANY KIND, either express or implied. See
+%% the License for the specific language governing rights and
+%% limitations under the License.
+%%
+%% The Original Code is RabbitMQ.
+%%
+%% The Initial Developer of the Original Code is GoPivotal, Inc.
+%% Copyright (c) 2007-2015 Pivotal Software, Inc.  All rights reserved.
+%%
+
+-module(rabbit_mqtt_test_util).
+
+-include_lib("eunit/include/eunit.hrl").
+
+%%--------------------------------------------------------------------
+%% coerce exchange and vhost to binaries
+%%--------------------------------------------------------------------
+
+%% coerce in case of string
+coerce_exchange_test() ->
+    <<"amq.direct">> = rabbit_mqtt_util:coerce(exchange, "amq.direct").
```

Comment:
sure, I will. While we are discussing tests, I want to share my thoughts about this task. Here I'm writing unit tests for a function that is (at least, so far) private (used only by the rabbit_mqtt_util:env/1 function). What I would have preferred is to call rabbit_mqtt_util:env/1 from my tests and check the return value using assertEqual. This is not simple, because env call application:get_env/2 and this would require an instance of rabbitmq running (I guess). My question is: can we use some stub/mock erlang library for this? Are we already using it in other repos?